### ISSUE TITLE
ActiveFedora::Base.uri_to_id doesn't properly handle long ids

### ISSUE THREAD

**Issue by jcoyne** [projecthydra/active_fedora#855]
```ruby
uri = "http://localhost:8983/fedora/rest/test/ff/36/55/73/ff365573w/files"
ActiveFedora::Base.uri_to_id(uri)
# => "files"
```
it should be `ff/36/55/73/ff365573w/files`.  This causes direct_containers to be created in the wrong place.

**Comment by jcoyne** [comment]
actually a bug in active_fedora-noid 0.2.0

**Comment by jcoyne** [comment]
Fixed in active_fedora-noid 0.3.0
